api/app/src/main/kotlin/packit/service/RoleService.kt

```diff
@@ -86,6 +91,27 @@ class BaseRoleService(
         rolePermissionService.removeRolePermissionsFromRole(role, removeRolePermissions)
     }
 
+    override fun getRoleNames(): List<String>
+    {
+        return roleRepository.findAll().map { it.name }
+    }
+
+    override fun getRolesWithRelationships(): List<Role>
+    {
+        return roleRepository.findAll()
+    }
+
+    override fun getRolesWithRelationships(isUsernames: Boolean): List<Role>
```

Comment:
Does "withRelationships" here just mean, 'including permissions'? ...oh, and users?

Reply:
have renamed as per above suggestion